apps/channels/models.py

```diff
@@ -155,6 +162,7 @@ def check_usage_by_another_experiment(platform: ChannelPlatform, identifier: str
         filter_params = {f"extra_data__{platform.channel_identifier_key}": identifier}
         channel = ExperimentChannel.objects.filter(**filter_params).first()
         if channel and channel.experiment != new_experiment:
+            # TODO: check if it's in a different team and if the user has access to that team
```

Comment:
Would we hide this link if the user doesn't have access, or what was your thoughts around this?